### PR TITLE
linkPhysicalAddress: do not verify external addresses

### PR DESCRIPTION
**Pull request by johnnyaug**

### CODE REVIEW

File: pkg/api/controller_test.go [hunk 1]
```diff
@@ -2437,6 +2434,38 @@ func TestController_LinkPhysicalAddressHandler(t *testing.T) {
 			t.Fatalf("LinkPhysicalAddress status code: %d, expected: %d", resp.HTTPResponse.StatusCode, expectedStatusCode)
 		}
 	})
+
```
Comment (N-o-Z) [inline pkg/api/controller_test.go:2437]: 🥇

Comment (N-o-Z) [reply pkg/api/controller_test.go:2437]: Maybe I missed it but can we add a test to try and link an expired physical address?

**Review by N-o-Z — CHANGES_REQUESTED** [review]
Looks good - though I have one concern
Shouldn't we deprecate the token field first and then remove it?

File: pkg/api/controller_test.go (hunk 1 above)
Comment (johnnyaug) [reply pkg/api/controller_test.go:2437]: I can't think of a trivial way to generate an expired address, so I think it would be out of scope for this PR.

Comment (johnnyaug) [reply pkg/api/controller_test.go:2438]: @N-o-Z - added the test

**Review by N-o-Z — APPROVED** [review]
LGTM! 
Thanks for adding the very much needed test coverage!

**Pull request merged**